libpod/sqlite_state.go

```diff
@@ -386,7 +386,7 @@ func (s *SQLiteState) ValidateDBConfig(runtime *Runtime) (defErr error) {
 		return nil
 	}
 
-	if err := checkField("os", os, runtimeOS); err != nil {
+	if err := checkField("dbOS", dbOS, runtimeOS); err != nil {
```

Comment:
This string seems to change a error message and hints to a non existing db field now